core/src/test/java/com/linecorp/armeria/server/AnnotatedHttpServiceRequestConverterTest.java

```diff
@@ -115,20 +123,51 @@ public String defaultText(@RequestObject String obj1) {
             return obj1;
         }
 
-        @Post("/default/invalid")
+        @Post("/default/javabean/{userName}/{seqNum}")
+        public String defaultJavaBeanForPost(@RequestObject RequestJavaBean1 bean1) throws JsonProcessingException {
+            assertThat(bean1).isNotNull();
+            assertThat(bean1.userName).isNotNull();
```

Comment:
should we confirm value of bean as well? (not just nullable check)

Reply:
Can not check value here, because value will be different for every request.

So I convert the whole request bean into a JSON string, and return it as a response, then check if the response is same as expected JSON string. like this:
`assertThat(response.content().toStringUtf8()).isEqualTo(expectedResponseContent)`